src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -1754,54 +1754,54 @@
         "CRON": "Cron"
     },
     "CLARITY": {
-        "OPEN": "Open",
-        "CLOSE": "Close",
-        "SHOW": "Show",
-        "HIDE": "Hide",
-        "EXPAND": "Expand",
+        "OPEN": "Ouvrir",
+        "CLOSE": "Fermer",
+        "SHOW": "Afficher",
+        "HIDE": "Cacher",
+        "EXPAND": "Etendre",
         "COLLAPSE": "Collapse",
-        "MORE": "More",
-        "SELECT": "Select",
-        "SELECT_ALL": "Select All",
-        "PREVIOUS": "Previous",
-        "NEXT": "Next",
-        "CURRENT": "Jump to current",
+        "MORE": "Plus",
+        "SELECT": "Sélectionner",
+        "SELECT_ALL": "Tout sélectionner",
+        "PREVIOUS": "Précédent",
+        "NEXT": "Suivant",
+        "CURRENT": "Passer au courant",
```

Comment:
Without context, I would say "Passer à l'actuel"

Reply:
In english it's `Jump to current`